drf_recaptcha/fields.py

```diff
@@ -7,12 +7,14 @@
 
 
 class ReCaptchaV2Field(CharField):
-    def __init__(self, **kwargs):
+    def __init__(self, secret_key: str = None, **kwargs):
```

Comment:
not documented, as I understand you add possibility to define the default key in field's args. let's add one more sample

```python
class DynamicContextSecretKey(APIView):
    def post(self, request):
        if request.platform == "android":
            recaptcha_secret_key = "SPECIAL_FOR_ANDROID"
        else:
            recaptcha_secret_key = "SPECIAL_FOR_IOS"

        serializer = WithReCaptchaSerializer(
            data=request.data,
            context={
                "request": request,
                "recaptcha_secret_key": recaptcha_secret_key,
            },
        )
        serializer.is_valid(raise_exception=True)
        ...


class DynamicContextSecretKey(GenericAPIView):
    serializer_class = WithReCaptchaSerializer

    def get_serializer_context(self):
        if self.request.platform == "android":
            recaptcha_secret_key = "SPECIAL_FOR_ANDROID"
        else:
            recaptcha_secret_key = "SPECIAL_FOR_IOS"

        context = super().get_serializer_context()
        context.update({"recaptcha_secret_key": recaptcha_secret_key})
        return context


class MobileSerializer(Serializer):
    recaptcha = ReCaptchaV3Field(secret_key="")
    ...
```

Reply:
create please also a new section similar to ### Priority of score value

### Priority of secret_key value

1.  settings `DRF_RECAPTCHA_SECRET_KEY`
2.  the argument `secret_key` of field
3.  request.context["recaptcha_secret_key"]